mm2src/coins/nft/storage/sql_storage.rs

```diff
@@ -883,36 +1195,67 @@ impl NftTransferHistoryStorageOps for SqliteNftStorage {
         .await
     }
 
-    async fn update_transfers_meta_by_token_addr_id(
+    async fn get_transfers_with_empty_meta(&self, chain: &Chain) -> MmResult<Vec<NftTokenAddrId>, Self::Error> {
```

Comment:
Nit suggestion: You don't need to deref this later if you pass it by value
```suggestion
    async fn get_transfers_with_empty_meta(&self, chain: Chain) -> MmResult<Vec<NftTokenAddrId>, Self::Error> {
```

Reply:
> Nit suggestion: You don't need to deref this later if you pass it by value

Yeah, since chain impls copy actually I can change param chain: &Chain in db methods to chain: Chain. 
I can do it for all db methods.

Reply:
But for example in `update_nft` function there are a lot of functions which use chain. chain enum currently is not big and it isnt changed during process, so rust compiler should optimize it pretty well. 
But still should we remove all references of chain?

Reply:
`*chain` already does the copy. When you read the function signature you see `&Chain`, but in fact, we are doing the copy by derefing it inside. So I was thinkig, why not directly pass the copy so we don't have this confusion.

> so rust compiler should optimize it pretty well

See https://github.com/KomodoPlatform/komodo-defi-framework/pull/1255#discussion_r864281984

Reply:
I changed to `chain: Chain` param in those storage methods in sql where there was *chain.

> See https://github.com/KomodoPlatform/komodo-defi-framework/pull/1255#discussion_r864281984

:eyes: